cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -250,19 +249,15 @@ def _process_static_resources(self, zip_src):
                 bundles.append(name)
 
         # Update package.xml
-        tree = ET.fromstring(package_xml)
-        section = tree.find(".//sf:types[sf:name='StaticResource']", self.namespaces)
-        if section is None:
-            section = ET.Element("{{{}}}types".format(self.namespaces["sf"]))
-            name = ET.Element("{{{}}}name".format(self.namespaces["sf"]))
-            section.append(name)
-            name.text = "StaticResource"
-            tree.find(".//sf:types[last()]", self.namespaces).addnext(section)
+        Package = metadata_tree.parse(package_xml)
+        sections = [typ for typ in Package.types if typ.name.text == "StaticResource"]
```

Comment:
We're doing this in a number of places. Would be nice if we could do something like `Package.find_by("types", name="StaticResource")`

Reply:
Sounds like you want an ORM. ;)

cd3dde5e

Reply:
I like the generalization to kwargs in findall.